r/R/utilities.R

```diff
@@ -0,0 +1,29 @@
+#
+# subset_ids subsets a seurat object with the cell ids
+#
+subset_ids <- function(scdata, cells_id) {
+  meta_data_subset <- scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
+  current_cells <- rownames(meta_data_subset)
+  scdata <- subset(scdata, cells = current_cells)
+  return(scdata)
+}
+
+insertSetChildThroughApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```

Comment:
instead of passing both `experiment_id` and `api_url` as parameters, you can import config in this file and directly use them from the imported config.